Dockerfile

```diff
@@ -74,42 +77,38 @@ RUN mkdir -p /userdata/cramCache/error
 
 RUN echo "cramRef=/userdata/cramCache" >> $APACHEDIR/cgi-bin/hg.conf
 
-RUN cd $APACHEDIR && git clone https://github.com/DailyDreaming/kentest2.git && cd kentest2 && git checkout pipeline-bog && cd .. && mv kentest2/ kent/
-#RUN cd $APACHEDIR && git clone https://github.com/DailyDreaming/kentest2.git && cd kentest2 && git checkout a && cd $APACHEDIR && mv kentest2/ kent/
-# RUN cp $APACHEDIR/kent/src/browserbox/usr/local/apache/cgi-bin/hg.conf $APACHEDIR/cgi-bin/hg.conf
+RUN cd /tmp && wget http://hgdownload.soe.ucsc.edu/admin/jksrc.zip && unzip jksrc.zip && rm -f jksrc.zip
+# This code is currently not compiling but eventually the below line
+# will replace the above jksrc.zip line
+#RUN cd /tmp && git clone https://github.com/DailyDreaming/kentest2.git && cd kentest2 && git checkout pipeline-bog && cd .. && mv kentest2/ kent/
+RUN cp /tmp/kent/src/browserbox/usr/local/apache/cgi-bin/hg.conf $APACHEDIR/cgi-bin/hg.conf
 
 ADD srcCompile.sh /tmp/srcCompile.sh
-RUN bash /tmp/srcCompile.sh
+RUN bash /tmp/srcCompile.sh || true
+RUN (crontab -l ;echo '* * * * * /tmp/kent/src/product/scripts/fetchCramReference.sh /userdata/cramRef/pending /userdata/cramRef /userdata/cramRef/error') | crontab -
 
-RUN echo '{"workspace": "scratch-lon", "workspace_namespace": "anvil-stage-demo", "copy_progress_indicator_type": "auto"}' > /var/www/.tnu_config
-RUN echo "cramRef=/userdata/cramCache" >> $APACHEDIR/cgi-bin/hg.conf
+#RUN echo '{"workspace": "scratch-lon", "workspace_namespace": "anvil-stage-demo", "copy_progress_indicator_type": "auto"}' > /var/www/.tnu_config
 
-RUN mkdir -p /usr/local/apache/trash
-RUN chmod 777 /usr/local/apache/trash
 
-RUN curl -SLO "https://github.com/just-containers/s6-overlay/releases/download/v2.2.0.3/s6-overlay-${ARCH}.tar.gz" \
-  && tar -xzf s6-overlay-${ARCH}.tar.gz -C / \
-  && tar -xzf s6-overlay-${ARCH}.tar.gz -C /usr ./bin \
-  && rm -rf s6-overlay-${ARCH}.tar.gz
+#RUN curl -SLO "https://github.com/just-containers/s6-overlay/releases/download/v2.2.0.3/s6-overlay-${ARCH}.tar.gz" \
+#  && tar -xzf s6-overlay-${ARCH}.tar.gz -C / \
+#  && tar -xzf s6-overlay-${ARCH}.tar.gz -C /usr ./bin \
+#  && rm -rf s6-overlay-${ARCH}.tar.gz
 
 COPY 00-mysql.sh /etc/cont-init.d/00-mysql.sh
 COPY 10-browser.sh /etc/cont-init.d/10-browser.sh
 
-RUN mkdir /etc/services.d/genomebrowserapache
-COPY genomebrowserapache/run /etc/services.d/genomebrowserapache/run
-RUN chmod 777 /etc/services.d/genomebrowserapache/run
+#RUN mkdir /etc/services.d/genomebrowserapache
+#COPY genomebrowserapache/run /etc/services.d/genomebrowserapache/run
+#RUN chmod 777 /etc/services.d/genomebrowserapache/run
 
 COPY ports.conf /etc/apache2/
 
 COPY index.html /usr/local/apache/htdocs/index.html
 
 EXPOSE 3306 33060 8000 8001 80 443 3333 873
 
-#ENTRYPOINT ["sh", "-c", "export GOOGLE_PROJECT=anvil-stage-demo; export WORKSPACE_NAME=scratch-lon; /etc/init.d/mysql start; apachectl -D FOREGROUND"]
-ENTRYPOINT ["/init"]
+ENTRYPOINT ["/bin/bash"]
```

Comment:
This should probably be changed back to `/init` with `s6-overlay`.

Reply:
I was getting an error about not finding /init, I'll add back the s6 stuff and see if it works, thanks